src/compo/viirs_l1bnc2ioda.py

```diff
@@ -0,0 +1,256 @@
+                                                                          np.array(solar_aa, dtype=np.float32))
+            self.outdata[('sensorZenithAngle', metaDataName)] = np.append(self.outdata[('sensorZenithAngle', metaDataName)],
+                                                                          np.array(sensor_za, dtype=np.float32))
+            self.outdata[('sensorAzimuthAngle', metaDataName)] = np.append(self.outdata[('sensorAzimuthAngle', metaDataName)],
+                                                                           np.array(sensor_aa, dtype=np.float32))
+
+            for iodavar in obsvars:
+                self.outdata[self.varDict[iodavar]['valKey']] = np.append(
+                    self.outdata[self.varDict[iodavar]['valKey']], np.array(vals, dtype=np.float32))
+                self.outdata[self.varDict[iodavar]['errKey']] = np.append(
+                    self.outdata[self.varDict[iodavar]['errKey']], np.array(errs, dtype=np.float32))
+                self.outdata[self.varDict[iodavar]['qcKey']] = np.append(
+                    self.outdata[self.varDict[iodavar]['qcKey']], np.array(qcfs, dtype=np.int32))
+
+        # Write other MetaData
+        output_chidx = np.array(channels, dtype=np.int32) - 1
+        self.outdata[('sensorCentralFrequency', metaDataName)] = np.array(frequency[output_chidx], dtype=np.float32)
+        self.varAttrs[('sensorCentralFrequency', metaDataName)]['units'] = 'Hz'
+        self.outdata[('sensorCentralWavelength', metaDataName)] = np.array(frequency[output_chidx], dtype=np.float32)
+        self.varAttrs[('sensorCentralWavelength', metaDataName)]['units'] = 'micron'
+        self.outdata[('sensorChannelNumber', metaDataName)] = np.array(channels, dtype=np.int32)
+
+        for tmpvar in ['solarZenithAngle', 'solarAzimuthAngle', 'sensorZenithAngle', 'sensorAzimuthAngle']:
+            self.varAttrs[(tmpvar, metaDataName)]['_FillValue'] = float_missing_value
+            self.varAttrs[(tmpvar, metaDataName)]['units'] = 'degrees'
+
+        DimDict['Location'] = len(self.outdata[('latitude', metaDataName)])
+        AttrData['Location'] = np.int32(DimDict['Location'])
+        DimDict['Channel'] = len(channels)
+        AttrData['Channel'] = np.int32(DimDict['Channel'])
+
+
+def main():
+
+    # get command line arguments
+    # Usage: python blah.py -i /path/to/obs/2021060801.nc /path/to/obs/2021060802.nc ... -t Analysis_time /path/to/obs/2021060823.nc
+    # -o /path/to/ioda/20210608.nc
+    # where the input obs could be for any desired interval to concatenated together. Analysis time is generally the midpoint of
+    # analysis window.
+    parser = argparse.ArgumentParser(
+        description=('Read NASA VIIRS M-band Level 1b file(s) and Converter'
+                     ' of native NetCDF format for observations of TOA reflectance'
+                     ' from VIIRS to IODA NetCDF format.')
```

Comment:
can you add in the description where the data should be downloaded from?

Reply:
Yes, I added them in.